src/_help/hubs/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Hubs
+---
+
+- TOC
+{:toc}
+
+## What is a hub
+
+Hubs are a feature available starting from our paid plans: they allow you to manage collections of API documentation simply and efficiently.
```

Comment:
What about moving the “attention paid plan only” paragraph into a formatted call-out (box in specific color) to make it stand out?

Reply:
👍